ui/admin/app/controllers/scopes/scope/auth-methods/auth-method/managed-groups/members.js

```diff
@@ -1,7 +1,7 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
 
-export default class ScopesScopeAuthMethodsAuthMethodManagedGroupsNewController extends Controller {
+export default class ScopesScopeAuthMethodsAuthMethodManagedGroupsMembersController extends Controller {
```

Comment:
This file needs to be nested under `../auth-method/managed-groups/managed-group/member.js`

Reply:
Updated. Good catch!